daft/analytics.py

```diff
@@ -74,7 +74,12 @@ def _post_segment_track_endpoint(payload: dict[str, Any]) -> None:
         },
         data=json.dumps(payload).encode("utf-8"),
     )
-    resp = urllib.request.urlopen(req)
+    resp = urllib.request.urlopen(
+        req,
+        # Timeout after 10 seconds
```

Comment:
I think we should do this even quicker maybe even 1 second? We want to fail really fast so that it doesn't affect usage performance

Reply:
Edited!